documentation/chatmeleon-swagger.yaml

```diff
@@ -283,6 +284,208 @@ paths:
                   type: string
                   format: uri
 
+  /friend/send:
```

Comment:
I've just remembered this concept of an idempotent API.
- What would happen if this endpoint gets called more than once? Return the same 200 success response? Return error, friend request already exists?
- What about if the friend request is already accepted but another friend request is sent?
- Friend request is declined, then send another friend request? (this I think would behave as a new friend request)

Small details but I think we should document this as well.